assets/src/styles/components/_hydrograph.scss

```diff
@@ -50,16 +50,24 @@
                 fill: black;
                 text-anchor: middle;
             }
-        }
-        .tick {
-            line {
-                fill: none;
-                stroke: lightgrey;
-                stroke-opacity: 0.7;
+            .tick {
+                line {
+                    fill: none;
+                    stroke: lightgrey;
+                    stroke-opacity: 0.7;
+                }
             }
-            text {
+        }
+        .x-axis {
```

Comment:
I found the x tick marks to be nearly invisible with using the lightgrey color. Made these black so they are easier to see.